### PR TITLE
Sass styles

### PR DESCRIPTION
**Pull request by bchucherko**
Frontpage styling:
1) conversion to scss
2) file restructuring
3) ui elements styled
4) easy pie chart script added - it's implemented on frontpage, however this requires some logic to make sense - right now there's no scale, so your free to modify it
5) custom icon fonts added to public folder - not sure if this is where it should go

### CODE REVIEW

**Comment by Nuanda** [comment]
@bchucherko Bartek, please set the yellow ready-for-review label (on the right hand panel) if you think this is ready to be merged. Before that we assume that you still want to play with this branch before merging.

File: app/assets/stylesheets/vendor/_select2.scss [hunk 1]
```diff
@@ -0,0 +1,535 @@
+.select2-container {
```
Comment (kammerer) [inline app/assets/stylesheets/vendor/_select2.scss]: File app/assets/stylesheets/select2.css should be removed since we have this one.

**Comment by kammerer** [comment]
I suppose is not ready to be merged yet but I took a peek and it mostly seems ok. Some things did not work for me though, e.g.:

* pie charts did not show
* footer sometimes covers page content
* "Next" button in submission is broken

File: app/assets/stylesheets/vendor/_select2.scss (hunk 1 above)
Comment (Nuanda) [inline app/assets/stylesheets/vendor/_select2.scss]: Fixed.

**Comment by Nuanda** [comment]
@kammerer I tried to fix the footer heights problem (it also caused the Next button being "hidden" behind footer's padding). I did it with something that is probably terrible - see https://github.com/eSpectrum-IT/brassica/commit/8b4ac6487d411044c8a64c1805bee6d2f131bfbe - but @bchucherko may fix it later. I just wanted submissions work again.

**Comment by Nuanda** [comment]
@bchucherko @kammerer Since I couldn't let that one wait until Bartosz comes back from his holidays, I decided to merge it to master anyway. I "kind of" fixed some problems, and I turn what Tomasz reported into issues: #165 #166

**Pull request merged**